miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/MenuController.java

```diff
@@ -280,7 +289,14 @@ public String constantsScript(HttpServletResponse response, final UriComponentsB
     createArray(mapper, baseUri, node, "librarySelections", libraryService.listLibrarySelectionTypes(), Dtos::asDto);
     createArray(mapper, baseUri, node, "libraryStrategies", libraryService.listLibraryStrategyTypes(), Dtos::asDto);
     createArray(mapper, baseUri, node, "libraryDesignCodes", libraryDesignCodeService.list(), Dtos::asDto);
-    createArray(mapper, baseUri, node, "platforms", platformService.list(), Dtos::asDto);
+    Set<Long> activePlatforms = sequencerService.list().stream().filter(sequencer -> sequencer.isActive())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)